src/views/Login.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="login-background">
+    <v-app>
```

Comment:
`v-app` should be used only once in the entire app, most likely in `App.vue`. Remove the `v-app` tag.

Reply:
You can just keep the root tag as `v-container` and remove `div.login-background` and `v-app` both in this component since this will get populated in `<router-view />` in `App.vue`